app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -386,17 +384,6 @@ protected void onActivityResult(int requestCode, int resultCode, Intent intent)
                     return;
                 }
                 break;
-            case UPGRADE_APP:
-                if (resultCode == RESULT_CANCELED) {
-                    return;
-                } else if(resultCode == RESULT_OK) {
```

Comment:
I can't remember. @ctsims / @wpride do either of you foresee any issues coming up if we apply an update without logging out and back in?